src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -59,101 +34,7 @@ GcsPlacementGroupScheduler::GcsPlacementGroupScheduler(
       gcs_resource_manager_(gcs_resource_manager),
       cluster_resource_scheduler_(cluster_resource_scheduler),
       raylet_client_pool_(raylet_client_pool),
-      ray_syncer_(ray_syncer) {
-  scheduler_strategies_.push_back(std::make_shared<GcsPackStrategy>());
```

Comment:
To be clear:
All placement group related strategies are moved out from `gcs_placement_group_scheduler.h/.cc` to `bundle_scheduling_policy.h/.cc`.